signer/src/utxo.rs

```diff
@@ -741,6 +784,55 @@ mod tests {
         }
     }
 
+    #[ignore = "For generating the SOLO_(DEPOSIT|WITHDRAWAL)_SIZE constants"]
+    #[test]
+    fn create_deposit_only_tx() {
+        // For solo deposits
+        let mut requests = SbtcRequests {
+            deposits: vec![create_deposit(123456, 30_000, 0)],
+            withdrawals: Vec::new(),
+            signer_state: SignerBtcState {
+                utxo: SignerUtxo {
+                    outpoint: generate_outpoint(550_000_000, 0),
+                    amount: 550_000_000,
+                    public_key: generate_x_only_public_key(),
+                },
+                fee_rate: 5.0,
+                public_key: generate_x_only_public_key(),
+                last_fees: None,
+            },
+            num_signers: 10,
+            accept_threshold: 2,
+        };
+        let keypair = Keypair::new_global(&mut OsRng);
+
+        let mut transactions = requests.construct_transactions().unwrap();
+        assert_eq!(transactions.len(), 1);
+
+        let mut unsigned = transactions.pop().unwrap();
+        testing::set_witness_data(&mut unsigned, keypair);
+
+        println!("Solo deposit vsize: {}", unsigned.tx.vsize());
```

Comment:
I guess this and the other print statement should be cleaned up before merging.

Reply:
No I want them in there. The purpose of this is to show the "max" virtual sizes of solo deposit-withdrawal BTC transactions.